docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -896,7 +896,7 @@ For a more detailed guide about debugging native images please refer to the xref
 
 == Using Monitoring Options
 
-Monitoring options such as JDK flight recorder, jvmstat, heap dumps, and remote JMX (experimental in Mandrel 23)
+Monitoring options such as JDK flight recorder, jvmstat, heap dumps, NMT, and remote JMX (experimental in Mandrel 23)
```

Comment:
```suggestion
Monitoring options such as JDK flight recorder, jvmstat, heap dumps, NMT (starting with Mandrel 24.1 for JDK 23), and remote JMX 
```

I think there is no reason to mention Mandrel 23.0 as the minimum supported version is now 23.1. It's probably worth mentioning that NMT is only available in >= 24.1 though.

Reply:
Good point!